### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by Ouail-01**
> ### In this pull request, I create the mobile website section where I list the portfolio projects.

# Learning objectives :
- Understand how to parse a Figma design to create a UI.
- Use Flexbox to place elements in the page.
- Use CSS Grid to place elements in the page.
- Build a personal portfolio site.

### CODE REVIEW

**Review by Yerimah — CHANGES_REQUESTED** [review]
# STATUS: INVALID REQUEST 🔴 

Hi @Ouail-01 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

for invalid pul request

## Invalid Code Review Request

You have submitted a project that doesn't contain a short descriptive summary of the changes introduced in the pull request. it is important to make your repo look professional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -59,6 +59,171 @@ <h2>I'm a software <br> developer</h2>
                     <img src="img/Disabled.png">
                 </a>
             </div>
-        </section>   
+        </section>
+        <!-- The work section -->
+        <section class="works-section">
+            <div class="text">
+                <h2>My Recent <br> Works</h2>
+                <hr>
```
Comment (joskalenda) [inline index.html:67]: - [ ] No comment @Ouail-01 , you have done an amazing job, I really appreciate it, your code is well structured, however, you have some few changes to do in the design before you project get approved, let's take a look them  🤗

oua1

- [ ] As highlighted, let me be more explicit.... at the first look on the background color of your app compared to the figma at the right, am sure to remark a difference between them, please make sure they match same as per the requirement
- [ ] the `multi-post` title, please use the same color and font as per the requirement.
- [ ] same as the technologies sections, kindly modify them to match the same design as in the figma

File: index.html [hunk 2]
```diff
@@ -59,6 +59,171 @@ <h2>I'm a software <br> developer</h2>
...
+                </div>
+            </div>
+        </section>  
+         <!-- About me section -->
+         <section class="aboutme-section">
+            <div class="about-me">
+                <img src="img/Mask Group.png" alt="bg-img">
+                <h3>About me</h3>
+                <p class="flex"> I can help you build a product, <br> feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    <br> don’t hestiate to contact me.</p>
```
Comment (joskalenda) [inline index.html:180]: - [ ] Same as here, please do these few changes, remember, it is recommanded you stcik on the figma design as much as posible by using the same `color, padding, font,..` all properties as used in the orignal design please.
- [ ] All your cards backgrounds are diffrent from the original design, kindly make sure you modify everything.
ouai2

File: index.html [hunk 3]
```diff
@@ -59,6 +59,171 @@ <h2>I'm a software <br> developer</h2>
...
+                        <img src="img/icon2.png" alt="icon2">
+                    </div>
+                    <h3>Frameworks</h3>
+                    <ul class="flex">
+                        <li>Boottrap</li>
+                        <li>Ruby on rails</li>
+                        <li>RSpec</li>
+                        <li>Capybara</li>
+                        <li>Selenium</li>
+                    </ul>
```
Comment (joskalenda) [inline index.html]: - [ ] Check this out, I would suggest you follow the design guidlines and update everything to match the correct figma template
fram

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @Ouail-01 , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]...Good design✅ 
- [ ]...Well structured code ✅ 
- [ ]...Green linters check✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @Ouail-01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Descriptive PR :heavy_check_mark: 
- Good UI design :fire: 
- Correct use of Github flow :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**